src/map/packet_system.cpp

```diff
@@ -6220,132 +6239,131 @@ void SmallPacket0x0DC(map_session_data_t* const PSession, CCharEntity* const PCh
     TracyZoneScoped;
     switch (data.ref<uint32>(0x04))
     {
-        case NFLAG_INVITE:
+        case 0x01:
```

Comment:
Can these magic numbers get enum'd, or some comments?

Reply:
There's already comments on these 1 line below the cases. Is that sufficient? I could move them up I suppose.

Reply:
Meh, leave it as is